toolbar/toolbar.js

```diff
@@ -319,9 +319,12 @@ var fullscreenUI;
  * @param highResIcons ; whether to use a high-resolution image for the icon
  */
 var initFullscreen = function(highResIcons) {
-    createButton('fullscreen', toggleFullscreen);
+    var id = 'fullscreen';
+    createButton(id, toggleFullscreen);
+    var button = $('#' + id);
```

Comment:
Worth making `createButton` return the button here, to save having to retrieve it via jQuery?